Backpack/Color/Classes/Generated/BPKColor.h

```diff
@@ -487,6 +487,34 @@ NS_SWIFT_NAME(Color) @interface BPKColor : NSObject
 */
 @property(nonatomic, class, readonly) UIColor * shadowXlColor __deprecated;
 
+/**
+ * The `textColorPrimaryLight` color from the Backpack palette.
+ *
+ * <div style="width: 100px; height: 100px; background: #111236ff; box-shadow: 0px 1px 3px 0px rgba(37,32,31,.3); border-radius: 6px;"></div>
+*/
+@property(nonatomic, class, readonly) UIColor * textColorPrimaryLight __deprecated;
```

Comment:
What's the story behind these extra ones that are deprecated?

Reply:
@k0nserv released them under the wrong name when he first added them - and then because they were released he decided they should stay but be deprecated properly.
They'll be released in the next major

Reply:
Ok, thanks

Reply:
Actually - these were never released on ios of android, so removing them isn't breaking!